src/components/Editor.jsx

```diff
@@ -47,6 +48,10 @@ class Editor extends React.Component {
     this.deEmphasize = this.deEmphasize.bind(this);
   }
 
+  shouldComponentUpdate(nextProps) {
+    return _.isEqual(nextProps, this.props)
```

Comment:
I do think we can utilize the `shouldComponentUpdate()` method to limit the number of re-renders, but compare specific props instead of the entire props object.